run-tests.pl

```diff
@@ -321,12 +321,6 @@ sub _run_test
       Future->wait_any(
          $f_test,
 
-         $loop->delay_future( after => 2 )
```

Comment:
Hmm.. I guess. I think I added that before adding the overall 10 second deadline, so it's probably less interesting now.

Fine, let it die.